examples/amp-story/advance-after-background-audio.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
```

Comment:
I think it's time to add these to our local `examples/` files, it's annoying having to copy paste them from our QA samples every time we test locally.

Reply:
Do you think you could try to keep them in an untracked folder for now? I'm afraid they'd get out of sync as we'd need to update the stories in two places, and that we might miss bugs because of it

Reply:
I don't think we'd have to change these at all, same as we expect users to not have to change their markup unless there's a breaking change. Having a simple access example like we do for other components/use cases in our `examples/` folder seems appropriate.